### ISSUE TITLE
Longer readiness timeout in admin chart

### ISSUE THREAD

**Issue by paulc4** [nuodb/nuodb-helm-charts#198]
Change `readinessTimeoutSeconds` to 5s (currently is 1) to match same value in database charts.

Teaching Temenos today we had exactly this issue. Extending the timeout fixed it.

**Comment by paulc4** [comment]
How does this get seen to?

**Comment by paulc4** [comment]
I was teaching Temenos Ops Center staff and helping them setup NuoDB on Azure and we were getting readiness timeouts trying to run the admin pod.  Aaron suggested using the longer timeout.  He has seen the same problem running on Open Shift.

The suggested value of 5 is admittedly arbitrary, choosing the same value as the database chart just seemed like a good place to start and it worked.  We have not evaluated a best value, but the current value is sometimes too small.

**Comment by mkysel** [comment]
NuoDB probes are using the `exec probe` which changed in K8s 1.20. https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

This explains why we haven't seen this issue in 1.15-1.19.

Here are the official docs:
```
Note:
Before Kubernetes 1.20, the field timeoutSeconds was not respected for exec probes: probes continued running indefinitely, even past their configured deadline, until a result was returned.

This defect was corrected in Kubernetes v1.20. You may have been relying on the previous behavior, even without realizing it, as the default timeout is 1 second. As a cluster administrator, you can disable the feature gate ExecProbeTimeout (set it to false) on each kubelet to restore the behavior from older versions, then remove that override once all the exec probes in the cluster have a timeoutSeconds value set.
If you have pods that are impacted from the default 1 second timeout, you should update their probe timeout so that you're ready for the eventual removal of that feature gate.

With the fix of the defect, for exec probes, on Kubernetes 1.20+ with the dockershim container runtime, the process inside the container may keep running even after probe returned failure because of the timeout.
```
